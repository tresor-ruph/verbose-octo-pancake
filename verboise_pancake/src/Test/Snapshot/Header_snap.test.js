import React from 'react'
import renderer from 'react-test-renderer'
import Header  from './../../layouts/header'

it('renders correctly', () => {
    const HeaderSnap = renderer.create(<Header />).toJSON();
    expect(HeaderSnap).toMatchSnapshot();

});