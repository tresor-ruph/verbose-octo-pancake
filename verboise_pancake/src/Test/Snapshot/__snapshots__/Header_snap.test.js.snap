// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="navbar navbar-expand-lg navbar-dark"
  style={
    Object {
      "backgroundColor": "#343A40",
      "height": "12vh",
      "left": "0px",
    }
  }
>
  <a
    className="navbar-brand"
    href="#"
    onClick={[Function]}
  >
    Verbose
  </a>
  <div
    className="mr-auto navbar-nav"
    onKeyDown={[Function]}
  />
  <div
    className="navbar-nav"
    onKeyDown={[Function]}
  />
</nav>
`;
